src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--tue calendar--month-length--28">
+      <div class="calendar__days calendar__days--day--1"></div>
```

Comment:
it would be better if you use `:nth-child` instead of modifying the HTML

Reply:
pay attention to comments, don't use modifiers for `n-th` block, use pseudo-classes instead